### PR TITLE
OCPBUGS-37973: [release-4.16]: Backport of sriov-network-metrics-exporter

### PR DESCRIPTION
**Pull request by zeeke**
Backport [sriov-network-metrics-exporter](https://github.com/k8snetworkplumbingwg/sriov-network-metrics-exporter) deployment and configuration.
Feature is Tech Preview in [4.16.z](https://issues.redhat.com/browse/CNF-4619)

### CODE REVIEW

**Comment by zeeke** [comment]
@SchSeba , @ajaggapa can you please take a look at this backport?

File: bundle/manifests/sriov-network-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -387,6 +387,20 @@ spec:
                   value: operator-webhook-service
                 - name: ADMISSION_CONTROLLERS_CERTIFICATES_INJECTOR_SECRET_NAME
                   value: network-resources-injector-secret
+                - name: METRICS_EXPORTER_IMAGE
+                  value: quay.io/openshift/origin-sriov-network-metrics-exporter:4.17
```
Comment (SchSeba) [inline bundle/manifests/sriov-network-operator.clusterserviceversion.yaml:391]: please update the version here

File: bundle/manifests/sriov-network-operator.clusterserviceversion.yaml [hunk 2]
```diff
@@ -387,6 +387,20 @@ spec:
...
+                - name: METRICS_EXPORTER_KUBE_RBAC_PROXY_IMAGE
+                  value: quay.io/openshift/origin-kube-rbac-proxy:4.17
```
Comment (SchSeba) [inline bundle/manifests/sriov-network-operator.clusterserviceversion.yaml:393]: please update the version here

File: config/manager/manager.yaml [hunk 3]
```diff
@@ -81,6 +81,20 @@ spec:
             value: operator-webhook-service
           - name: ADMISSION_CONTROLLERS_CERTIFICATES_INJECTOR_SECRET_NAME
             value: network-resources-injector-secret
+          - name: METRICS_EXPORTER_IMAGE
+            value: quay.io/openshift/origin-sriov-network-metrics-exporter:4.17
```
Comment (SchSeba) [inline config/manager/manager.yaml:85]: please update the version here

File: config/manager/manager.yaml [hunk 4]
```diff
@@ -81,6 +81,20 @@ spec:
...
+          - name: METRICS_EXPORTER_KUBE_RBAC_PROXY_IMAGE
+            value: quay.io/openshift/origin-kube-rbac-proxy:4.17
```
Comment (SchSeba) [inline config/manager/manager.yaml:87]: please update the version here

**Review by SchSeba — APPROVED** [review]
/lgtm
/approve

**Pull request merged**